cumulusci/tasks/bulkdata/extract.py

```diff
@@ -335,3 +342,27 @@ def _sqlite_dump(self):
         with open(path, "w", encoding="utf-8") as f:
             for line in self.session.connection().connection.iterdump():
                 f.write(line + "\n")
+
+    def append_filter_clause(self, soql, filter_clause):
+        """Function that applies filter clause to soql if it is defined in mapping yml file"""
+
+        if not filter_clause:
+            return soql
+
+        # If WHERE keyword is specified in the maping file replace it with empty string.
+        # match WHERE keyword only at the start of the string and whitespace after it.
+        if filter_clause:
+            filter_clause = re.sub(
+                pattern=r"^WHERE\s+",
+                repl="",
+                string=filter_clause.strip(),
+                flags=re.IGNORECASE,
+            )
```

Comment:
We can remove the if statement here, because we already returned on line 349 if filter_clause was not truthy.

```suggestion
        filter_clause = re.sub(
            pattern=r"^WHERE\s+",
            repl="",
            string=filter_clause.strip(),
            flags=re.IGNORECASE,
        )
```

btw, thank you for using keyword arguments with re.sub; it makes it easier to read.